src/tool/GenerateAction.cpp

```diff
@@ -74,12 +74,38 @@ DoGenerateAction()
     // Load the compilation database file
     //
     // --------------------------------------------------------------
-    MRDOCS_CHECK(toolArgs.inputPaths, "The compilation database path argument is missing");
-    MRDOCS_CHECK(toolArgs.inputPaths.size() == 1,
-        formatError(
-            "got {} input paths where 1 was expected",
-            toolArgs.inputPaths.size()));
-    auto compilationsPath = files::normalizePath(toolArgs.inputPaths.front());
+    if (detail::failed(toolArgs.inputPaths) && detail::failed(toolArgs.generateCompilationDatabase))
```

Comment:
`detail::failed` is not meant to be used like that. It's an abstraction for `MRDOCS_CHECK/TRY` so they can work with any type. You just want to use `if (x.empty())` here.